### DISCUSSION TITLE
SBAS take time problems

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#643] [General]
@Xiaohua-Eric-Xu I have a question. There is a clear sense that SBAS analysis is becoming more demanding on memory size and taking longer than before. Is there any modification to the code? For the same data, SBAS analysis could produce results in one day before, but now it takes half a month to complete.

**Comment by Xiaohua-Eric-Xu** [comment]
It really depends on your problem size and what you feed into sbas. The solution time is not linear with number of acquisitions so may increase with time rapidly. This is also a reason why many space agencies (like LicSAR or ARIA) only produce sequential or nearest two or three interferograms.
